src/breadcrumb-group/item/funnel.tsx

```diff
@@ -3,29 +3,38 @@
 import React from 'react';
 import clsx from 'clsx';
 
-import { DATA_ATTR_FUNNEL_KEY, FUNNEL_KEY_FUNNEL_NAME } from '../../internal/analytics/selectors';
+import {
+  DATA_ATTR_FUNNEL_KEY,
+  DATA_ATTR_RESOURCE_TYPE,
+  FUNNEL_KEY_FUNNEL_NAME,
+} from '../../internal/analytics/selectors';
 
 import analyticsSelectors from '../analytics-metadata/styles.css.js';
-import styles from './styles.css.js';
 
 interface FunnelBreadcrumbItemProps {
+  className?: string;
   text: string;
-  last: boolean;
-  hidden?: boolean;
-  ghost?: boolean;
+  itemIndex: number;
+  totalCount: number;
+  disableAnalytics?: boolean;
 }
 
 export const FunnelBreadcrumbItem = React.forwardRef<HTMLSpanElement, FunnelBreadcrumbItemProps>(
```

Comment:
Refactored `FunnelBreadcrumbItem` to allow us rendering "no-UI" breadcrumbs we use to collect funnel metrics